pom.xml

```diff
@@ -199,15 +190,15 @@
 			<version>4.4.13</version>
 		</dependency>
 		<dependency>
-		    <groupId>net.sf.jasperreports</groupId>
-		    <artifactId>jasperreports</artifactId>
-		    <version>6.17.0</version>
-		    <exclusions>
-		    	<exclusion>
-		    		<groupId>com.lowagie</groupId>
-		    		<artifactId>itext</artifactId>
-		    	</exclusion>
-		    </exclusions>
+			<groupId>net.sf.jasperreports</groupId>
+			<artifactId>jasperreports</artifactId>
+			<version>6.16.0</version>
+			<exclusions>
+				<exclusion>
+					<groupId>com.lowagie</groupId>
+					<artifactId>itext</artifactId>
+				</exclusion>
+			</exclusions>
```

Comment:
the correct version is 6.17 as latest bump

Reply:
updated the current version